src/commands/report.ts

```diff
@@ -55,7 +55,7 @@ export async function execute(interaction: CommandInteraction) {
   try {
     submissionInteraction = await interaction.awaitModalSubmit({
       filter: (i) => i.customId === "reportModal" && i.user.id === user.id,
-      time: 60_000,
+      time: 240_000,
```

Comment:
timeout update here. 
After 240 seconds of the modal being open, a discord error will be thrown notifying that the report took too long to be filled out.

Reply:
```suggestion
      time: 4 * 60 * 1000, // 4 minutes
```